extensions/wikia/UserLogin/js/UserLoginSpecial.wikiamobile.js

```diff
@@ -1,10 +1,35 @@
-$(function(){
-	var msgBox = document.getElementById('wkLgnMsg'),
-		msg = msgBox && msgBox.innerText;
-
-	if(msg){
-		require('toast', function(t){
-			t.show(msg);
-		});
+require(['jquery', 'wikia.window', 'wikia.loader', 'toast'], function ($, window, loader, toast) {
+	'use strict';
+
+	function loadFacebookLoginScripts() {
+		loader({
+			type: loader.LIBRARY,
+			resources: 'facebook'
+		}, {
+			type: loader.MULTI,
+			resources: {
+				messages: 'fblogin',
```

Comment:
This line doesn't seem to be necessary since we're already loading the message in FacebookButtonController.

We could remove `JSMessages::enqueuePackage( 'fblogin', 'inline' );` from the controller and move  `JSMessages::registerPackage( 'fblogin' , array( 'wikiamobile-facebook-connect-fail' ) );` to UserLogin.setup.php so that it would be available asynchronously like this, but then we'd be registering that package for every request to MW. I say we just remove this line.